woocommerce/woocommerce
# Remove install support for Woo.com hosted products

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes the ability to auto install Woo.com hosted products and replaces the feature with a redirect to Woo.com. WP.org hosted products can still be automatically installed.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout [this Woo.com PR](https://github.com/Automattic/woocommerce.com/pull/19091)
2. Checkout this PR
3. Connect your local site to the local Woo.com site
4. Confirm in dev tools that subscript API responses include `is_installable` data
5. Click on install of a Woo.com hosted product and confirm it redirects you to the Woo.com installer (it's ok if the install fails)
6. Click on install of a WP.org hosted product ([MailChimp for WooCommerce](https://woocommerce.test/products/mailchimp-for-woocommerce/?utm_source=subscriptionsscreen&utm_medium=product&utm_campaign=wcaddons&utm_content=product-name)) and confirm it installs the product



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Replace installer with redirect for Woo.com hosted products.

</details>
